libcodechecker/analyze/analyzers/result_handler_plist_to_stdout.py

```diff
@@ -48,6 +48,20 @@ def print_steps(self, value):
         """
         self.__print_steps = value
 
+    @property
+    def print_report_hash(self):
+        """
+        Print the hash of the report.
+        """
+        return self.__print_report_hash
```

Comment:
We should set a default value to this.
Like here:
https://github.com/Ericsson/codechecker/blob/a3d8f812c236bedfcc89a5cb331156e4a4d66ab4/libcodechecker/analyze/analyzers/result_handler_plist_to_stdout.py#L32